### PR TITLE
Added the `-f` flag to change the foreground color

### PR DESCRIPTION
**Pull request by anthonysterling**
I've added the `-f` flag enabling the selection of the following colors:-

- white
- yellow
- black
- magenta
- blue
- green
- red

By default the foreground is blue, as per the current implementation. I did at the `-b` for background color selection but it looked a bit odd so dropped it.

This is not a language I know, so please go easy on me. I am however, looking forward to your feedback. 😄

### CODE REVIEW

File: src/nms.c [hunk 1]
```diff
@@ -284,3 +284,43 @@ char getMaskChar(void) {
...
+/*
+ * char getColorByName(char *string, int fallback)
+ *
+ * DESCR:
+ * Returns an ncurses color by its name.
+ *
+ */
+int getColorByName(char *string, int fallback) {
+
+	if(strcmp("white", string) == 0) {
```
Comment (anthonysterling) [inline src/nms.c]: A hashmap of some sort would be better here, is there a nice way to do this?

File: src/nms.h [hunk 2]
```diff
@@ -15,9 +16,12 @@ typedef struct {
 	int input_cursor_y;
 	bool show_cursor;
 	bool auto_decrypt;
+	int foreground_color;
 } NmsArgs;
 
 // Display the characters stored in the display queue
 char nms_exec(NmsArgs *);
 
+int  getColorByName(char *string, int fallback);
```
Comment (anthonysterling) [inline src/nms.h]: I'm not really sure if this should be here.

**Comment by bartobri** [comment]
Yeah, overall it works well, but I am making some tweaks to the code before I merge it. I think I'd like to pass the the raw color string (blue, red, green, etc...) to nms_exec() and let it do the converting with /getColorByName()/ internally. I think this will make for slightly cleaner code. We can probably drop the /int fallback/ parameter too, and simply return to COLOR_BLUE if all strcmps fail.

**Comment by bartobri** [comment]
See my commit 9ab70d19 if you're curious about the changes I made.

**Comment by anthonysterling** [comment]
Yep, all that makes sense. Thank you @bartobri.

**Pull request merged**